server/consumer.go

```diff
@@ -477,7 +486,7 @@ func (mset *Stream) AddConsumer(config *ConsumerConfig) (*Consumer, error) {
 		}
 		// If we are here we have already registered this durable. If it is still active that is an error.
 		if eo.Active() {
-			return nil, fmt.Errorf("consumer already exists and is still active")
+			return nil, fmt.Errorf("consumer already 1exists and is still active")
```

Comment:
typo

Reply:
Was fat fingering 'goto line 1' in emacs alot last night :/